### DISCUSSION TITLE
Is there a way to generate point plot with color scale?

### DISCUSSION THREAD

**Discussion by colorizer** [kookma/ogpf#26] [Q&A] [answered]
Hi, Is it possible to plot point data with color scale (based on the value in the 3rd column) similar to https://codeyarns.com/tech/2015-06-19-how-to-plot-points-in-color-in-gnuplot.html
![image](https://user-images.githubusercontent.com/64630824/136844573-01cd7571-a786-4832-86ac-e95a7db5b47f.png)

If not, is there a way to add another data column to the output .gp file generated by ogpf?
Thanks in advance!

**Comment by kookma** [comment]
I think it is possible! To do so I need an example in gnuplot and I then help you produce it in ogpf.
Note that, ogpf is quite flexible. It is a single file with few subroutines.

It gets data from Fortran, create a temp file and pipe it to gnuplot!

**Comment by kookma** [comment, marked as answer]
More info
The `gp%option()` is quite powerful and you can pass any command to gnuplot! It is like you run gnuplot from command window!

**Comment by colorizer** [comment]
Yes, thank you! I've written the data to separate file and called `gp%option` to plot those data! I wonder if its possible to append the fortran data to the same temp file being created by ogpf instead of me saving it separately. But, it's not really necessary tbh!

**Reply by kookma** [reply]
from your fortran program write the data in a temp file and use it as you did by calling `pg%option`
